docs/src/artifacts.md

```diff
@@ -0,0 +1,177 @@
+# [**8.** Artifacts](@id Artifacts)
+
+`Pkg` can install and manage containers of data that are not Julia packages.  These containers can contain platform-specific binaries, datasets, text, or any other kind of data that would be convenient to place within an immutable, life-cycled datastore.
+These containers, (called "Artifacts") can be created locally, hosted anywhere, and automatically downloaded and unpacked upon installation of your Julia package.
+This mechanism is also used to provide the binary dependencies for packages built with [`BinaryBuilder.jl`](https://github.com/JuliaPackaging/BinaryBuilder.jl).
+
+## `Artifacts.toml` files
+
+`Pkg` provides an API for working with artifacts, as well as a TOML file format for recording artifact usage in your packages, and to automate downloading of artifacts at package install time.
+Artifacts can always be referred to by content hash, but are typically accessed by a name that is bound to a content hash in an `Artifacts.toml` file that lives in a project's source tree.
+An example `Artifacts.toml` file is shown here as an example:
+
+```TOML
+# Example Artifacts.toml file
```

Comment:
```suggestion
```

Reply:
I would rather have that there, so that users that are scanning through quickly can see that (a) comments in TOML file works, and (b) they know the filename of this file.